static/app/actionCreators/guides.tsx

```diff
@@ -56,6 +65,37 @@ export function recordFinish(guide: string, orgId: string | null) {
     },
   });
 
+  let tour = '',
```

Comment:
@roggenkemper definitely need some comments about why this is needed (aka used in the sandbox). I'd like to move the sandbox functionality to a separate function we call here if possible. The goal is to separate the code as much as possible that's practical.